### ISSUE TITLE
Create Content Block: PDF displayer/embed

### ISSUE THREAD

**Issue by wwwwwlwwwww** [distributeaid/distributeaid.org#586]
This is one of a new set of issues being created to break down https://github.com/distributeaid/distributeaid.org/issues/523 into a larger number of smaller tasks to make sure that our new contributors have plenty of things they can choose to work on. https://github.com/distributeaid/distributeaid.org/issues/523 has some additional context for the information below.

We would like to be able to display PDFs inline on our website.  The only real parameter here that I can think of would be the PDF file itself.  

For the first iteration of this, it is probably prudent to just focus on getting a pdf to display one page at a time with a scroll functionality to move through the pages.

Eventually, we would like to make it so that documents can be viewed as booklets, kind of like this: https://issuu.com/distraid/docs/distribute_aid_quarterly_report_q2_2021.   Not sure if this is easier to code up or use an embed of some sort (the issue with the website I linked was that it's a paid service).  Anyway, you should NOT worry about making the PDF viewer have booklet functionality unless it jumps out as straightforward and quick to do.

**Comment by coderbyheart** [comment]
> We would like to be able to display PDFs inline on our website.

Why?

PDFs are not meant to be displayed as a webpage, because they are statically formatted content optimized for printing. What is wrong with using a PDF viewer?

**Comment by jtfairbank** [comment]
@coderbyheart agreed, might be nice to have a "preview" style thing tho?  Like if we want to feature a pdf somewhere with more then just a link or button, but not display the whole thing.  I'll sync w/ @wwwwwlwwwww on this one.
